grails-app/assets/javascripts/streama/controllers/modal-media-detail-ctrl.js

```diff
@@ -54,5 +66,5 @@ angular.module('streama').controller('modalMediaDetailCtrl', [
 
 		$scope.$on('$stateChangeStart', function () {
 			$uibModalInstance.dismiss('cancel');
-		})
+		});
```

Comment:
this is a very large commit... the original commits were 5, I believe. why did you choose to make it all as one commit?